package.json

```diff
@@ -62,7 +62,8 @@
   },
   "betterScripts": {
     "docs": "jsdoc -c jsdoc.json && node ./scripts/jsdoc-inject-dist.js",
-    "lint": "prettier test/**.js src/**.js --write && eslint ./ && tslint -c tslint.json src/index.d.ts",
+    "lint": "./scripts/prettier-test.sh && eslint ./ && tslint -c tslint.json src/index.d.ts",
+    "lint:fix": "prettier test/**.js src/**.js --write",
```

Comment:
`lint:fix` should probably also run `eslint --fix` after prettier... Either directly, or by using [prettier-eslint](https://github.com/prettier/prettier-eslint)